session.go

```diff
@@ -0,0 +1,57 @@
+package proxy
+
+import (
+	"io"
+
+	"github.com/emersion/go-smtp"
+)
+
+type session struct {
+	c  *smtp.Client
+	be *Backend
+}
+
+func (s *session) Reset() {
+	s.c.Reset()
+}
+
+func (s *session) Send(from string, to []string, r io.Reader) error {
+	if err := s.Mail(from); err != nil {
```

Comment:
Err, I missed this, but `Send` is not needed anymore

Reply:
Okay, just created PR #3.